### PR TITLE
Comparison cleanup

### PR DESCRIPTION
**Pull request by cytadela8**
- more verbose comparison function
- adjusted bytes and bytecode comparison for consistency
- fixed oracle query type

### CODE REVIEW

**Comment by cytadela8** [comment]
By the way AFAIK nothing uses this `aeb_fata_data:lt` function. It is needed for my maps_to_list work

File: src/aeb_fate_data.erl [hunk 1]
```diff
@@ -237,19 +240,19 @@ format_kvs(List) ->
...
 ordinal(T) when ?IS_FATE_INTEGER(T)   -> ?ORD_INTEGER;
 ordinal(T) when ?IS_FATE_BOOLEAN(T)   -> ?ORD_BOOLEAN;
-ordinal(T) when ?IS_FATE_ADDRESS(T)   -> ?ORD_ADDRESS;
-ordinal(T) when ?IS_FATE_CHANNEL(T)   -> ?ORD_CHANNEL;
-ordinal(T) when ?IS_FATE_CONTRACT(T)  -> ?ORD_CONTRACT;
-ordinal(T) when ?IS_FATE_ORACLE(T)    -> ?ORD_ORACLE;
-ordinal(T) when ?IS_FATE_BYTES(T)     -> ?ORD_BYTES;
+ordinal(T) when ?IS_FATE_ADDRESS(T)   -> ?ORD_ADDRESS;%TODO
+ordinal(T) when ?IS_FATE_CHANNEL(T)   -> ?ORD_CHANNEL;%TODO
+ordinal(T) when ?IS_FATE_CONTRACT(T)  -> ?ORD_CONTRACT;%TODO
```
Comment (radrow) [inline src/aeb_fate_data.erl:245]: What is "todo" here?

Comment (cytadela8) [reply src/aeb_fate_data.erl:245]: Fixed

**Comment by cytadela8** [comment]
Replaced by #98

**Pull request closed without merging**